src/components/categoryitems.main.less

```diff
@@ -1,5 +1,101 @@
 @import './../style/common.less';
 
+.products-container {
+  position: relative;
+
+  .loading-product {
+    .category-items-listing {
+      position: relative;
+
+      &:before {
+        content: "";
+        height: 100%;
+        width: 100%;
+        background: rgba(255,255,255,0.8);
+        position: absolute;
+        top: 0;
+        left: 0;
+        z-index: 2;
+      }
+    }
+  }
+
+  .loader {
```

Comment:
This loader styling will need to be applied to the general styles to take effect on all pages.